docs/plugins/filters/memcached.asciidoc

```diff
@@ -111,7 +111,7 @@ filter {
   * Value type is <<hash,hash>>
   * There is no default value for this setting.
 
-If specified, extracts the values from the given event fields, and sets the corresponding keys to those values in memcached with the configured <<plugins-{type}s-{plugin}-ttl,ttl>>
+If specified, extracts the values from the given event fields, and sets the corresponding keys to those values in memcached with the configured <<#plugins-{type}s-{plugin}-ttl,ttl>>
```

Comment:
memcached 1.0.1 includes these fixes (required for our conversion to asciidoctor).  Is updating the lock file to 1.0.1 or adding a commit with the fixes here the right approach?